plugins/jobtype/src/azkaban/jobtype/HadoopSecureHiveWrapper.java

```diff
@@ -51,35 +50,21 @@
   private static final String SINGLE_QUOTE_STRING = Character
       .toString(SINGLE_QUOTE);
 
-  private static boolean securityEnabled;
   private static final Logger logger = Logger.getRootLogger();
 
   private static CliSessionState ss;
   private static String hiveScript;
 
   public static void main(final String[] args) throws Exception {
-    
-    Properties props = HadoopSecureWrapperUtils.loadAzkabanProps();
-    HadoopConfigurationInjector.injectResources(new Props(null, props));
 
-    hiveScript = props.getProperty("hive.script");
+    Properties jobProps = HadoopSecureWrapperUtils.loadAzkabanProps();
+    HadoopConfigurationInjector.injectResources(new Props(null, jobProps));
 
-    final Configuration conf = new Configuration();
-
-    UserGroupInformation.setConfiguration(conf);
-    securityEnabled = UserGroupInformation.isSecurityEnabled();
-
-    if (HadoopSecureWrapperUtils.shouldProxy(props)) {
-      UserGroupInformation proxyUser = null;
-      String userToProxy = props.getProperty("user.to.proxy");
-      if (securityEnabled) {
-        proxyUser = HadoopSecureWrapperUtils.createSecurityEnabledProxyUser(userToProxy, logger);
-      } else {
-        proxyUser = UserGroupInformation.createRemoteUser(userToProxy);
-      }
-
-      logger.info("Proxied as user " + userToProxy);
+    hiveScript = jobProps.getProperty("hive.script");
 
+    if (HadoopSecureWrapperUtils.shouldProxy(jobProps)) {
```

Comment:
Indeed there's a slight duplication of check.

I've originally wrote the code to depend on the return value of setupProxyUser(), but I thought that might not be as readable as duplicating that function call twice